### PR TITLE
[stable/rabbitmq-ha] Fixed invalid value

### LINKED ISSUE #18463: [stable/rabbitmq-ha] Invalid default value in values.yaml
**Describe the bug**
The `values.yaml` file has the word `None` where it should be an empty value.

**Version of Helm and Kubernetes**:
helm 2.14.3, k8s 1.14.8

**Which chart**:
rabbitmq-ha

**What happened**:
When you set `service.type: LoadBalancer` you get this error:
```
Error: UPGRADE FAILED: Service "chartname-rabbitmq-ha" is invalid: spec.clusterIP: Invalid value: "None": may not be set to 'None' for LoadBalancer services
```

**Anything else we need to know**:
It took me a while to track down, but the answer here is to set in your own variables `service.clusterIP` to an empty value.  But, the chart should probably be fixed - set this same variable to an empty value in the chart-provided `values.yaml` file, it's on line 13.

### PR DESCRIPTION
**Pull request by austinorth**
#### What this PR does / why we need it:
Sets the `service.clusterIP` variable to an empty value and commented out, following the standard set in other charts for optional empty values, to avoid invalid value error. Also updated readme and bumped chart version.

Signed-off-by: Austin Orth <aorth@niche.com>

#### Which issue this PR fixes
  - fixes #18463

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[stable/mychartname]`)

**CC:** @steven-sheehy

### CODE REVIEW

File: stable/rabbitmq-ha/values.yaml [hunk 1]
```diff
@@ -315,7 +315,7 @@ terminationGracePeriodSeconds: 10
 
 service:
   annotations: {}
-  clusterIP: None
+  # clusterIP:
```
Comment (Gsantomaggio) [inline stable/rabbitmq-ha/values.yaml]: I would leave `clusterIP`,  without the comment
Or maybe better `clusterIP: null`

File: stable/rabbitmq-ha/Chart.yaml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 name: rabbitmq-ha
 apiVersion: v1
 appVersion: 3.8.0
-version: 1.36.0
+version: 1.37.0
```
Comment (Gsantomaggio) [inline stable/rabbitmq-ha/Chart.yaml]: `version` should be `1.36.1` since is a minor fix

**Review by steven-sheehy — CHANGES_REQUESTED** [review]
This is a major breaking change. `.spec.clusterIP` is an immutable field and this changes the default from `None` to "", breaking current deployments that use the default value of `None`. The upgrade would fail and they'd have to manually delete the service for it to succeed. I'm inclined not to merge this.

If you'd like to specifically fix the referenced issue by disallowing `None` when type LoadBalancer, I would consider that.

File: stable/rabbitmq-ha/values.yaml (hunk 1 above)
Comment (austinorth) [reply stable/rabbitmq-ha/values.yaml]: Reverting to the original value after @steven-sheehy 's comment

**Comment by Gsantomaggio** [comment]
@austinorth you should resolve the conflicts. ty!

**Review by steven-sheehy — APPROVED** [review]
/lgtm
Thanks!

**Pull request merged**